src/hooks/useRebalanceHandler.tsx

```diff
@@ -21,6 +29,10 @@ export const useRebalanceHandler = () => {
   const [slippage, setSlippage] = useState<string>('')
   const [slippageMinutes, setSlippageMinutes] = useState<string>('')
 
+  const [posManager, setPosManager] = useState<Contract | null>(null)
+  const [gammaPool, setGammaPool] = useState<Contract | null>(null)
```

Comment:
why do we need gamma pool now? should use everything from posmgr.

Reply:
In rebalance I need cfmm address. You said that I can get it from .cfmm() in GammaPool that's why I am using it

Reply:
ok